Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -1,58 +1,31 @@
 const SteamScript = include("engines.wine.quick_script.steam_script");
-
-const DotNET472 = include("engines.wine.verbs.dotnet472");
-const Vcrun2017 = include("engines.wine.verbs.vcrun2017");
+const dotnet472 = include("engines.wine.verbs.dotnet472");
+const vcrun2017 = include("engines.wine.verbs.vcrun2017");
```

Comment:
I think we should use a uniform notation here. It gets quite confusing if some scripts use `dotnet472`, others `DotNET472` or `Dotnet472` etc. This also makes it harder to search for occurrences of a script or plugin.

@plata does it make sense to change the verbs, plugins etc. to return the verb not as a default export but as a named export, e.g.:

```javascript
module.DotNET472 = ...
```

This will enforce other people to import the script via:

```javascript
val { DotNET472 } = include("engines.wine.verbs.dotnet472");
```

Reply:
Are there cases, where such a behavior exists? E.g. Lua modules would be similar to what we have now. For me the current behavior makes sense because otherwise you must now the internals of the verb to be able to use it.

Reply:
We will have issues with the current approach if we decide to perform larger refactoring later on. For example if we decide to change the `DotNET472` verb by adding a new input which must be set with `withX(...)` it becomes much easier in my opinion if you can simply search for all occurrences for `DotNet472`.